Makefile

```diff
@@ -8,6 +8,7 @@ IMAGE_ORG?=openshift-cert-manager
 IMAGE_TAG?=latest
 IMAGE_OPERATOR?=$(IMAGE_REGISTRY)/$(IMAGE_ORG)/cert-manager-operator:$(IMAGE_TAG)
 IMAGE_OPERATOR_BUNDLE?=$(IMAGE_REGISTRY)/$(IMAGE_ORG)/cert-manager-operator-bundle:$(IMAGE_TAG)
+TESTS_IMAGE?=$(IMAGE_REGISTRY)/$(IMAGE_ORG)/cert-manager-scorecard-tests:$(IMAGE_TAG)
```

Comment:
Can we preserve the image naming convention - `cert-manager-operator-scorecard-test`?